src/application/editor.py

```diff
@@ -27,6 +29,10 @@ class ThoughtBox(MenuNav):
     def __init__(self):
         # Create notes directory
         os.makedirs(NOTES_DIR, exist_ok=True)
+        # If welcome page isn't present, create it.
+        if not os.path.isfile(NOTES_DIR + "/" + WELCOME_PAGE):
+            copyfile(ASSETS_DIR + WELCOME_PAGE, NOTES_DIR + "/" + WELCOME_PAGE)
```

Comment:
```
copyfile(os.path.join(ASSEST_DIR, WELCOME_PAGE), os.path.join(NOTES_DIR, WELCOME_PAGE))
```
Mostly just for consistency with other code that uses `os.path.join`, since all these paths are constants. Your call.